tests/extensions/magiclink_strip_http.html

```diff
@@ -0,0 +1,110 @@
+<h3>Should Pass</h3>
```

Comment:
I really think we don't need to copy the entire test file just to unit test the stripping of the protocol.  You should be able to effectively show it is working with a couple of examples.